src/lib/hooks/use-seam-client.ts

```diff
@@ -0,0 +1 @@
+export { useSeamClient } from 'lib/seam/use-seam-client.js'
```

Comment:
Hmm, why the duplication of export locations for this and `use-devices`?

Reply:
Ok, so you would like to add an entrypoint accessible via `@seamapi/react/hooks`. Can do! https://github.com/seamapi/react/issues/101